infra/vsts_agent_linux.tf

```diff
@@ -18,7 +18,7 @@ resource "google_compute_region_instance_group_manager" "vsts-agent-linux" {
   name               = "vsts-agent-linux"
   base_instance_name = "vsts-agent-linux"
   region             = "${local.region}"
-  target_size        = 10
+  target_size        = 6
```

Comment:
Any particular reason why we are shrinking the target size? Just because it isn’t necessary given our current load?

Reply:
Yes; I've been looking at utilization over the last week and have never seen more than 6 concurrent Windows jobs running. There have sometimes been a bit more Linux jobs (I've seen 17 running + queued at one point), but they dissolve quickly back to 6 or less (this is because of the "other" Linux jobs: collect build data, check standard change label, etc.).

It's also part of the experiment for sizing the macOS pool: if we switch to 6 machines for Linux & Windows and nobody complains it means 6 would also be a good size for the macOS one.